### PR TITLE
1142 Update status badges in README

### LINKED ISSUE #1142: Fix readme status badges
**What Needs to be Done?**

Use new badges for Azure Pipelines

### PR DESCRIPTION
**Pull request by lifflander**
Fixes #1142

### CODE REVIEW

**Comment by lifflander** [comment]
Waiting for https://github.com/DARMA-tasking/vt/tree/1142-fix-status-badges to show properly

**Comment by lifflander** [comment]
It's annoying that all the badges have the same name now... I don't see anyway to change that in the badge generation code.

Screen Shot 2020-11-10 at 2 51 26 PM

File: README.md [hunk 1]
```diff
@@ -4,18 +4,18 @@
...
-<!-- ![gcc-8, ubuntu, mpich, address sanitizer](https://github.com/DARMA-tasking/vt/workflows/PR%20tests%20(gcc-8,%20ubuntu,%20mpich,%20address%20sanitizer)/badge.svg?branch=develop) -->
-<!-- ![gcc-9, ubuntu, mpich, zoltan](https://github.com/DARMA-tasking/vt/workflows/PR%20tests%20(gcc-9,%20ubuntu,%20mpich,%20zoltan)/badge.svg?branch=develop) -->
-<!-- ![gcc-10, ubuntu, openmpi, no LB](https://github.com/DARMA-tasking/vt/workflows/PR%20tests%20(gcc-10,%20ubuntu,%20openmpi,%20no%20LB)/badge.svg?branch=develop) -->
-<!-- ![clang-3.9, ubuntu, mpich](https://github.com/DARMA-tasking/vt/workflows/PR%20tests%20(clang-3.9,%20ubuntu,%20mpich)/badge.svg?branch=develop) -->
-<!-- ![clang-5, ubuntu, mpich, trace](https://github.com/DARMA-tasking/vt/workflows/PR%20tests%20(clang-5.0,%20ubuntu,%20mpich,%20trace)/badge.svg?branch=develop) -->
+[![gcc-5, ubuntu, mpich](https://dev.azure.com/DARMA-tasking/DARMA/_apis/build/status/PR%20tests%20(gcc-5%2C%20ubuntu%2C%20mpich)?branchName=develop)](https://dev.azure.com/DARMA-tasking/DARMA/_build/latest?definitionId=2&branchName=develop)
```
Comment (JacobDomagala) [inline README.md:7]: You can specify badge's description by using Label tag. For example here you can add `&Label=(gcc-5%2C%20ubuntu%2C%20mpich)` to first link, after `?branchName=develop`. So entire badge is as follows:

[gcc-5, ubuntu, mpich?branchName=develop&Label=(gcc-5%2C%20ubuntu%2C%20mpich))](https://dev.azure.com/DARMA-tasking/DARMA/_build/latest?definitionId=2&branchName=develop)

**Review by JacobDomagala — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lifflander) [reply README.md]: Thanks! That's super helpful!

**Review by cz4rs — APPROVED** [review]

**Comment by cz4rs** [comment]
image

**Review by jstrzebonski — APPROVED** [review]

**Comment by jstrzebonski** [comment]
LGTM, although I wonder if we could't split all those badges one in each line? They all seem to blur together at the moment..

**Pull request merged**